shell/platform/embedder/tests/embedder_test_context.cc

```diff
@@ -69,6 +69,10 @@ const std::string& EmbedderTestContext::GetAssetsPath() const {
   return assets_path_;
 }
 
+const std::string& EmbedderTestContext::GetApplicationLibraryPath() const {
+  return application_library_path_;
```

Comment:
this is currently an empty string because the text fixtures are currently set up to only produce elf snapshots and load AOT data that way. I'm not sure how to configure tests to build application libraries, or if that would even be appropriate. At least for the purposes of the test I'm adding, the empty string is sufficient